ui/package.json

```diff
@@ -18,28 +18,30 @@
     "preview": "vite preview --port 5002"
   },
   "dependencies": {
+    "@featherds/radio": "^0.10.13",
```

Comment:
Should we add this to dev dependencies instead?

Reply:
yes, and not only this... Pinia, axios, router?...  dev dependecies or peerDependecies?

Reply:
Yup Pinia and the router could be dev dependencies. Hmm, not sure about axios, opennms isn't setup to share that so I would leave it for now. Do the calls work when used as a plugin?